Views/index.erb

```diff
@@ -16,5 +16,19 @@ if false is an easy way to comment out a large block of code.
 <% end %>
 	<div ng-controller="greetingController">
 		{{greeting}}
+	
+	<script type="text/ng-template" id="tree_item_renderer.html">
```

Comment:
The trick here is that this template is specified by its `id` value. It's then ng-included recursively. This is really a super hack, but I suppose it gives you something to fuck with in terms of recursive angular views. (I don't like it myself but meh)